### PR TITLE
update docker image url to ECR

### PR DESCRIPTION
**Pull request by serventera**

### CODE REVIEW

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -114,23 +114,45 @@ jobs:
...
-        labels: ${{ steps.meta.outputs.labels }}
+    - name: Login to Amazon ECR
+      if: github.event_name != 'pull_request' && (github.ref == 'refs/heads/main' || contains(github.ref, 'refs/tags/'))
+      id: login-ecr
+      uses: aws-actions/amazon-ecr-login@v1
+
+    - name: Build, tag, and push image to Amazon ECR
+      if: github.event_name != 'pull_request' && (github.ref == 'refs/heads/main' || contains(github.ref, 'refs/tags/'))
+      env:
+        ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
```
Comment (mochadwi) [inline .github/workflows/docker-image.yml:140]: ada list property yg available apa aja kah mas, dari ecr github action nya? 

Kalo `steps.login-ecr.outputs.labels` ada ndak yah? @serventera

Comment (mochadwi) [reply .github/workflows/docker-image.yml:140]: Alternatif yg line 158 mas

**Review by mochadwi — APPROVED** [review]
Lgtm, thanks a lot yaaa mas @serventera

File: .github/workflows/docker-image.yml [hunk 2]
```diff
@@ -114,23 +114,45 @@ jobs:
...
+    #   with:
+    #     context: .
+    #     # Android SDK not support arm64 yet
+    #     # platforms: linux/amd64,linux/arm64
+    #     platforms: linux/amd64
+    #     # platforms: linux/arm64
+    #     # push: ${{ github.event_name != 'pull_request' && github.ref == 'refs/heads/main' }}
+    #     push: ${{ github.event_name != 'pull_request' && (github.ref == 'refs/heads/main' || contains(github.ref, 'refs/tags/')) }}
+    #     tags: sampingan/android:latest,${{ steps.meta.outputs.tags }}
+    #     labels: ${{ steps.meta.outputs.labels }}
```
Comment (mochadwi) [inline .github/workflows/docker-image.yml:158]: It's okay to remove this imho. As we already use the ECR Github Actions

Comment (mochadwi) [reply .github/workflows/docker-image.yml:158]: Idk if the `platforms` attributes required for ECR Github action or only used for Hub Docker Github Action cmiiw? Is it okay to omit the attributes `platforns` mas?

**Review by mochadwi — COMMENTED** [review]
a few feedback added

File: .github/workflows/docker-image.yml (hunk 2 above)
Comment (serventera) [reply .github/workflows/docker-image.yml:158]: klo di ecr platform itu optional bal. soale di aws-actions/amazon-ecr-login@v1 g ada property platform, tags, labels gitu. jadi setelah login berhasil langsung build terus push

**Review by sakti — APPROVED** [review]
LGTM, for initial step

**Pull request merged**